src/chuck_absyn.cpp

```diff
@@ -102,6 +102,15 @@ a_Stmt_List prepend_stmt_list( a_Stmt stmt, a_Stmt_List stmt_list, int pos )
     return a;
 }
 
+a_Stmt_List append_stmt_list( a_Stmt_List stmt_list, a_Stmt stmt, int pos )
+{
+    a_Stmt_List a = new_stmt_list( stmt, pos );
+    a->next = stmt_list;
```

Comment:
The function signature accepts the statment_list as the first argument and the statement as the second argument, as specified in the `chuck.y` file.

The body of this function was simply copied over from the previous function `prepend_stmt_list`. Should append `a` to `stmt` by finding the first `NULL` next value in the linked list and assign `a`?